liquibase/scripts/generate_liquibase_snapshots.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public License,
+#  v. 2.0. If a copy of the MPL was not distributed with this file, You can
+#  obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+#  the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#
+#  Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+#  graphic logo is a trademark of OpenMRS Inc.
+#
+
+
+function generate_liquibase_snapshots_and_update() {
+	cd liquibase/
+	. scripts/create_liquibase_snapshots.sh "${1}" "${2}"
+	. scripts/fix_liquibase_snapshots.sh
+	cd ..
+}
+
+function move_updated_snapshots() {
+    mv liquibase/snapshots/liquibase-core-data-UPDATED-SNAPSHOT.xml api/src/main/resources/org/openmrs/liquibase/snapshots/core-data/"liquibase-core-data-$openmrs_version.x.xml"
+    mv liquibase/snapshots/liquibase-schema-only-UPDATED-SNAPSHOT.xml api/src/main/resources/org/openmrs/liquibase/snapshots/schema-only/"liquibase-schema-only-$openmrs_version.x.xml"
+}
+
+function create_new_liquibase_update_file() {
+    new_liquibase_file="liquibase-update-to-latest-${new_openmrs_version}.x.xml"
+    cp "api/src/main/resources/liquibase-update-to-latest-template.xml" "api/src/main/resources/org/openmrs/liquibase/updates/$new_liquibase_file"
+    
+    database_updater_class="api/src/test/java/org/openmrs/util/DatabaseUpdaterDatabaseIT.java"
+    variable_name="CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X"
+    
+    # Increase the variable value by 1
+    sed -i "s/\($variable_name *= *\)[0-9]\+/\1$(( $(grep -o "$variable_name *= *[0-9]\+" "$database_updater_class" | awk '{print $NF}') + 1 ))/" "$database_updater_class"
+    
+    insert_line="	<include file=\"org/openmrs/liquibase/updates/$new_liquibase_file\"/>"
+    sed -i -e '$!N;$!N;$i\'"$insert_line" "api/src/main/resources/liquibase-update-to-latest-from-1.9.x.xml"
```

Comment:
Why are we using 1.9.x here?

Reply:
if the liquibase snapshot update version is above 1.9.x , we add that names in this file. This also mentioned in [here](https://github.com/openmrs/openmrs-core/tree/master/liquibase#step-2---create-a-new-liquibase-update-file)

Reply:
In the documentation that you referenced above, isn't 1.9.x just an example. Where the actual value depends on the current version?

Reply:
The file called liquibase-update-to-latest-from-1.9.x.xml is used for integration tests. When we make a new snapshot, we create a file for the next snapshot version using the format liquibase-update-<snapshot-next-version>.x.xml. After creating this new file, we need to add its path to the liquibase-update-to-latest-from-1.9.x.xml file.